src/components/CarouselCard/__tests__/animationButton.spec.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import AnimationButtom from '../animationButton';
+
+describe('<AnimationButtom />', () => {
+    it('should set the right assistive text', () => {
+        const component = mount(<AnimationButtom isAnimationPaused />);
+
+        expect(component.find('span').text()).toBe('Start auto-play');
+    });
+
+    it('should set the right aria-pressed value', () => {
+        const component = mount(<AnimationButtom isAnimationPaused />);
+
+        expect(component.find('button').prop('aria-pressed')).toBe(true);
```

Comment:
here the same, you don't know anything about ButtonIcon details, only check that the ButtonIcon has the ariaPressed prop set to true.